src/TSMapEditor/UI/Windows/TriggersWindow.cs

```diff
@@ -1725,11 +1813,16 @@ private void LbActionParameters_SelectedIndexChanged(object sender, EventArgs e)
 
                 tbActionParameterValue.Text = GetParamValueText(triggerAction.Parameters[paramNumber], triggerParamType, triggerActionParam.PresetOptions);
                 tbActionParameterValue.TextColor = GetParamValueColor(triggerAction.Parameters[paramNumber], triggerParamType);
+
+                var isSupportedGoToSourceParamType = supportedGoToSourceTriggerParamTypes.Contains(triggerParamType);
```

Comment:
I don't like hiding of types of primitives. As explained in the contribution guide.

```suggestion
                bool isSupportedGoToSourceParamType = supportedGoToSourceTriggerParamTypes.Contains(triggerParamType);
```